pkg/models/storage/storage.go

```diff
@@ -27,6 +28,11 @@ import (
 	"kubesphere.io/kubesphere/pkg/informers"
 )
 
+const (
+	IsDefaultStorageClassAnnotation     = "storageclass.kubernetes.io/is-default-class"
```

Comment:
Unnecessary exported constants

Reply:
OK, I will fix it.

Reply:
IsDefaultStorageClassAnnotation is used at PatchStorageClass.